2.15/2.15.7/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 2.15.7
+В самом начале выводится 2 пары сообщений:
+1) "Введите имя" и "Введите возраст"
+2) "Введите имя" и "Введите возраст"
+
+А потом вывести 2 алерта с инфой из введенных данных:
+"Этого человека зовут - Х. Он [еще молод], потому что ему Y лет."
+Причем фраза [еще молод] может поменяться на [уже взрослый], если введенный для него возраст был больше 18.
+<script>
+	function userInformation(name, age) {
+		let ageText = 'еще молод';
+
+		if (age > 18) {
```

Comment:
![image](https://github.com/tacya9/JS-1-2/assets/115877101/c5527b1d-6ed4-4401-b7b1-4de15b38f395)
this block can be rewritten to one line, how can you do it?